assets/src/edit-story/components/canvas/layout.js

```diff
@@ -233,6 +233,14 @@ const NavNextArea = styled(NavArea).attrs({
   area: 'f',
 })``;
 
+const QuickActionsArea = styled(PaddedPage)`
+  grid-area: p;
+  position: absolute;
+  right: calc(-24px + var(--page-padding-px));
+  padding-left: 0;
+  padding-right: 0;
+`;
```

Comment:
Works for now. While this is being reviewed I'm going to update the grid so that this fits in the right spot.

Reply:
Moved to another ticket: #7510